internal/provider/resource_cloudflare_waiting_room_event.go

```diff
@@ -42,19 +42,21 @@ func expandWaitingRoomEvent(d *schema.ResourceData) (cloudflare.WaitingRoomEvent
 		return cloudflare.WaitingRoomEvent{}, err
 	}
 
-	prequeueStartTime := time.Time{}
+	var prequeueStartTime *time.Time
 	if t, ok := d.GetOk("prequeue_start_time"); ok {
-		prequeueStartTime, err = time.Parse(time.RFC3339, t.(string))
+		prequeueStartTimeValue, err := time.Parse(time.RFC3339, t.(string))
+		prequeueStartTime = &prequeueStartTimeValue
```

Comment:
we also have `Bool` and `BoolPtr` helper methods if you'd like to use them instead? 

https://github.com/cloudflare/cloudflare-go/blob/9af8b925a93059adb1840ab07c5dbc79c1a428fb/convert_types.go#L75-L86